cogs/moderation.py

```diff
@@ -16,12 +27,24 @@ def __init__(self, bot):
         with open('banned.txt', 'r', encoding="utf-8") as f:
             self.banned = f.read().splitlines()
 
+    def remove_invites(self, message):
+        isInvite = re.search(r'\b(?:https?:\/\/)?(?:www\.)?discord\.gg\/\w+\b', message)
+        if isInvite:
+            logger.info(f'Link {isInvite.group()} in message {message}')
+            message.delete()

```

Comment:
testing this didn't work for me.  It caused the following error:

```
Traceback (most recent call last):
  File "C:\Python310\lib\site-packages\discord\client.py", line 441, in _run_event  
    await coro(*args, **kwargs)
  File "c:\Users\renat\Documents\GitHub\ArseneWenger\cogs\moderation.py", line 47, in on_message
    self.remove_invites(msgLower)
  File "c:\Users\renat\Documents\GitHub\ArseneWenger\cogs\moderation.py", line 34, in remove_invites
    message.delete()
AttributeError: 'str' object has no attribute 'delete'
```

I see that lower on the line you have an `await message.delete()` that may work better, but i can't even get to that part.  I added the await command and made the function async but that didn't work either.